src/create_disk.sh

```diff
@@ -382,7 +383,6 @@ x86_64)
         --boot-directory $rootfs/boot \
         $disk
     fi
-    install_grub_cfg
```

Comment:
This has the side-effect of swapping the order of `cp grub.cfg` and `grub-install`, would that have actual implications on the bootloader bits on disk?

Reply:
I don't think so, this just installs to the MBR.  The config file is read dynamically after mounting `/boot` at runtime.